.pre-commit-config.yaml

```diff
@@ -25,7 +25,7 @@ repos:
           ]
         additional_dependencies: ["colorama"]
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.5.0
+    rev: v3.0.0-alpha.1
```

Comment:
@YiyangLi this files don't belong to this change. Please revert this.

Reply:
good catch, reverted.